### PR TITLE
This is all i got so far

### PR DESCRIPTION
**Pull request by Waruggy**

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -63,22 +63,22 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log('s{inventory[32].car_year}', '${inventory[32].car_make}', '#{inventory[32].car_model}');
```
Comment (Zealll) [inline assignments/arrays.js]: What is `s` and `#` ?

File: assignments/arrays.js [hunk 2]
```diff
@@ -63,22 +63,22 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+console.log('Lincoln','Town Car');
```
Comment (Zealll) [inline assignments/arrays.js]: You are supposed to get inside of the array and grab it from there, not hard code it

File: assignments/arrays.js [hunk 3]
```diff
@@ -63,22 +63,22 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+console.log(carModels.sort());
```
Comment (Zealll) [inline assignments/arrays.js]: You are actually console logging an empty array here

File: assignments/arrays.js [hunk 4]
```diff
@@ -63,22 +63,22 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+console.log(new ca);
```
Comment (Zealll) [inline assignments/arrays.js]: I don't know what you are console logging here

File: assignments/objects.js [hunk 5]
```diff
@@ -18,21 +18,55 @@ const example = {
...
+    "email": "adaine5@samsung.com",
+    "gender": "F"
+}
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(int1.name)= "Mitzi"
```
Comment (Zealll) [inline assignments/objects.js]: We don't do equal signs `(=)` next to our console logs, JavaScript doesn't work like that

**Comment by Waruggy** [comment]
I updated.

**Pull request merged**